implants/lib/eldritch/src/lib.rs

```diff
@@ -1,24 +1,33 @@
+#[allow(clippy::needless_lifetimes)]
+pub mod assets;
+#[allow(clippy::type_complexity)]
+#[allow(clippy::needless_lifetimes)]
 pub mod file;
+#[allow(clippy::type_complexity)]
+#[allow(clippy::needless_lifetimes)]
+#[allow(clippy::too_many_arguments)]
+pub mod pivot;
+#[allow(clippy::needless_lifetimes)]
 pub mod process;
+#[allow(clippy::needless_lifetimes)]
+#[allow(clippy::type_complexity)]
```

Comment:
Oh my.
Not sure what's going on here I think this is kinda confusing to have added.
Do we want to just disable those features in the clippy check instead of adding all these execption blocks to our code?

Reply:
experimentation is needed methinks